src/radio-button.jsx

```diff
@@ -15,9 +15,11 @@ let RadioButton = React.createClass({
   },
 
   propTypes: {
+    checkedIcon: React.PropTypes.element,
```

Comment:
I think that we should use `node` that is less restrictive.
Could you add this to the documentation?